Atualizando Cocoapods + Fixes

BrunoMazzo opened the pull request:
Baixei o projeto aqui e não consegui compilar. Daí corrigi o que foi necessário para ele rodar. Basicamente as coisas feitas foram:
- Atualizar o GemFile.lock com as novas versões (cocoapods 1.0.1 e suas dependências)
- Alteração do podfile
  - Não foi alterado nada de importante, foi mais uma tentativa de fazer a build funcionar
- Corrigir o Other_CFlags para usar $(inherited)
  - Sem isso a flag $PODS_ROOT não era criada e a build não funcionava
- Aproveitei para criar um script de setup
  - Ele basicamente instala o Bundler, suas dependências, o Swiftlint e roda o Pod Install
  - Atualizei o README para usar esse script

BrunoMazzo committed:
Updating Gemfile.lock

BrunoMazzo committed:
Update for cocoapod 1.0.1

BrunoMazzo committed:
Readding R.generated to correct path

BrunoMazzo committed:
Creating a setup script to install all dependencies

- Install Bundler, Homebrew, Swiftlint and run cocoapods

barbosa commented on setup.sh:
```diff
@@ -0,0 +1,26 @@
+
+echo "Executing CocoaPods"
+cd CocoaHeadsApp
+pod install
```
De repente aqui seja melhor voltar para a pasta anterior após a execução do `pod install`. Só por questões de consistência. O que acha?

Talvez tudo num comando só: `cd CocoaHeadsApp && pod install && cd -`

BrunoMazzo committed:
Returning for the Root Folder before Setup

- Returning to project root folder before setup finish
- Executing Cocoapods on a single command

BrunoMazzo commented:
É uma boa idéia mesmo. Acabei de subir a alteração.

barbosa commented:
Valeu @BrunoMazzo !

🍻
